app/src/main/java/ch/blinkenlights/android/vanilla/PlaybackService.java

```diff
@@ -1174,18 +1174,19 @@ private void scrobbleBroadcast() {
 			intent.putExtra("track", song.title);
 			intent.putExtra("album", song.album);
 			intent.putExtra("artist", song.artist);
-			if (androidIds[0] != -1) {
-				intent.putExtra("songid", androidIds[0]);
```

Comment:
Why were those deleted?

Reply:
My bad , no need to remove for scrobbleBroadcast()